libs/render_delegate/utils.cpp

```diff
@@ -144,6 +144,21 @@ auto nodeSetVec2FromVec2d = [](AtNode* node, const AtString paramName, const GfV
     AiNodeSetVec2(node, paramName, static_cast<float>(v[0]), static_cast<float>(v[1]));
 };
 
+auto nodeSetRGBFromVec4 = [](AtNode* node, const AtString paramName, const GfVec4f& v) {
+    AiNodeSetRGB(node, paramName, v[0], v[1], v[2]);
+};
+auto nodeSetRGBFromVec4d = [](AtNode* node, const AtString paramName, const GfVec4d& v) {
+    AiNodeSetRGB(node, paramName, static_cast<float>(v[0]), static_cast<float>(v[1]), static_cast<float>(v[2]));
+};
+
+auto nodeSetRGBAFromVec3 = [](AtNode* node, const AtString paramName, const GfVec3f& v) {
+    AiNodeSetRGBA(node, paramName, v[0], v[1], v[2], 1.f);
```

Comment:
are we sure we always want 1.f ?

Reply:
One test on linux is failing

Reply:
this is what the implicit conversion in Arnold does when you convert color3 to color4, alpha is set to 1